src/main/java/seedu/address/logic/Logic.java

```diff
@@ -18,22 +18,36 @@
 public interface Logic {
     /**
      * Executes the command and returns the result.
+     *
      * @param commandText The command as entered by the user.
      * @return the result of the command execution.
      * @throws CommandException If an error occurs during command execution.
-     * @throws ParseException If an error occurs during parsing.
+     * @throws ParseException   If an error occurs during parsing.
      */
     CommandResult execute(String commandText) throws CommandException, ParseException;
 
+
     /**
-     * Exports properties to csv file.
+     * Executes the command and returns the result.
+     *
+     * @param commandText The command as entered by the user.
+     * @param file        File chosen by the user.
```

Comment:
Is this argument description alignment consistent with the rest of the comments?
What do the others think?

Reply:
can refer to this https://github.com/nus-cs2103-AY2122S1/forum/issues/10

Reply:
Reverted to old style.